src/app/layouts/base-layout/components/header/nav-item/nav-item.module.scss

```diff
@@ -52,6 +52,11 @@
       width: 16px;
       height: 17px;
       margin-left: 10px;
+
+      > img {
```

Comment:
Do we really need `>` ?

Reply:
No, but I like to be very specific about styles. In this case, it's the `img` that follows the `span`, not any `img` inside it.